README.md

```diff
@@ -35,6 +35,6 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - To my coding partners
-- Microverse Team
-- Standup Call Team
-- Morning call Team
+- To microverse Team
```

Comment:
- [ ] [OPTIONAL] Please ensure your README file has all the links working. This exudes professionalism in your work. Please fix the link to the issues page.